public/app/kanbanize/controllers/BoardController.js

```diff
@@ -0,0 +1,52 @@
+angular.module('app')
+	.controller('BoardController', ['$scope', '$log', '$stateParams', 'kanbanizeService', 'itemService', '$mdToast', '$state',
+		function ($scope, $log, $stateParams, kanbanizeService, itemService, $mdToast, $state) {
+		$scope.boardSetting = {
+				"projectId" : kanbanizeService.getProjectId()
+			};
+			$scope.ITEM_STATUS = itemService.ITEM_STATUS;
+			kanbanizeService.getBoardDetails($stateParams.orgId, $stateParams.boardId, 
+				function(data) {
+					$scope.mapping = data.mapping;
+					$scope.boardId = $stateParams.boardId;
+					$scope.boardSetting.streamName = data.streamName || kanbanizeService.getStreamName();
+					angular.forEach(data.mapping, function(value, key) {
```

Comment:
Perché fai questa operazione? La struttura che hai in mapping non va bene così com'è?